itest/rust/build.rs

```diff
@@ -280,6 +338,164 @@ fn generate_rust_methods(inputs: &[Input]) -> Vec<TokenStream> {
         .collect()
 }
 
+struct PropertyTemplate {
+    rust: TokenStream,
+    gdscript: String,
+}
+
+fn generate_property_template(inputs: &[Input]) -> PropertyTemplate {
+    let mut rust = Vec::new();
+    let mut gdscript = Vec::new();
+    gdscript.push(String::from(
+        "class_name PropertyTemplateGDScript extends Node\n",
```

Comment:
Is the `class_name` needed here? It adds an extra step for refactorings (file rename), so if not important to the test, I'd omit it.